suite-native/transactions/src/components/TransactionsList/TransactionList.tsx

```diff
@@ -65,6 +67,38 @@ type TransactionListItem =
     | (TypedTokenTransferWithTx | MonthKey)
     | (WalletAccountTransaction | MonthKey);
 
+type TransactionsListFooterProps = {
+    isLoading: boolean;
+    canShowButton: boolean;
+    onButtonPress: () => void;
+};
+
+const TransactionsListFooter = ({
```

Comment:
This file is already too long. Could you please move this component to a separate file?

Reply:
10127190bd